frontend/src/index.ts

```diff
@@ -723,6 +726,54 @@ export class App extends LiteElement {
       `,
     });
   }
+
+  private startSyncBrowserTabs() {
+    // TODO remove this line after this change has been deployed
+    // for more than 24 hours
+    window.localStorage.removeItem(AuthService.storageKey);
```

Comment:
This is just to ease in migrating existing clusters, right?

Reply:
Yep!